src/DGtal/topology/doc/moduleCubicalComplex.dox

```diff
@@ -308,6 +308,71 @@ Example topology/cubical-complex-collapse.cpp shows that it works also in 3D (in
 @image html cubical-complex-illustrations-collapse.png "Homotopic collapse of \a X, displayed in orange, with fixed points marked in blue."
 @image latex cubical-complex-illustrations-collapse.png "Homotopic collapse of \a X, displayed in orange, with fixed points marked in blue." width=5cm
 
+@section dgtal_ccomplex_sec8 Thinning in cubical complexes
+
+Thinning of digital objects in 2D and 3D with the guarantee that a final skeleton is thin can be achieved directly
+only in \f$\mathbb{Z}^2\f$ for so-called well-composed images. Nevertheless, the problem can be solved for 3D objects
+represented by cubical complexes @cite Chaussard:IWCIA:09.
+
+In @cite Chaussard:IWCIA:09 Chaussard and Couprie proposed a parallel directional thinning schemes in cubical complexes
+ which allow to obtain a thin result. The implementation in DGtal works for both 2D and 3D cubical complexes.
+
+The first scheme, so-called ParDirCollapse---parallel directional collapse---based on the idea such that free pairs
+of faces are collapsed with respect to a fixed order of directions and order of face dimensions. In other words, in
+each iteration we first collapse free paris of a given direction starting with pairs of the highest dimension.
+When there is no more free pairs to remove for a given direction then another direction is considered until specified
```

Comment:
until the specified number of iterations is reached ?
or 
unless the specified number of iterations is not reached ?